src/Wannabe Edgerunner/r6/scripts/WannabeEdgerunner/track/LocKeyChecker.reds

```diff
@@ -2,19 +2,46 @@ import Edgerunning.System.EdgerunningSystem
 import Edgerunning.Common.E
 
 public class EdgerunnerInteractionChecker extends ScriptableEnv {
-  let sleep: String = "LocKey#46418";
+  let sleep: array<String>;
   let shower: String = "LocKey#46419";
   let donate: String = "LocKey#45546";
   let pet: array<String>;
   let apartment: array<String>;
+  let lover: array<String>;
+  let social: array<String>;
 
   private cb func OnLoad() {
+    this.sleep = [
+	  "LocKey#46047",  // Interacting with lover while in bed, or as part of the sleep action when they are over.
+	  "LocKey#46418"   // bed
+	];
     this.pet = [ 
       "LocKey#39016", // Iguana
       "LocKey#32348", // Cat
       "LocKey#32183", // Nibbles
-      "LocKey#32185"  // Nibbles
+      "LocKey#32185", // Nibbles
+	  "LocKey#80033", // Cat, near Pacifica Rollercoaster
+	  "Elmo", // Deceptious Cat Petting Mod Support
+	  "Nali", // Deceptious Cat Petting Mod Support
+	  "Leia", // Deceptious Cat Petting Mod Support
+	  "Jack", // Deceptious Cat Petting Mod Support
+	  "Suzi", // Deceptious Cat Petting Mod Support
+	  "Nibbles" // Deceptious Cat Petting Mod Support
     ];
+	this.social = [
+	  "LocKey#46442",  // Rollercoaster
+	  "LocKey#48683",  // Rollercoaster, "raise hands" in excitement.
+	  "LocKey#49986",  // Dance interaction; Triggers on dance start in vanilla. Triggers on selecting partner and going back in menu w/ Deceptious mods.
+	  "LocKey#42930",  // Sitting down to drink or standing up to stop. Triggers on selecting partner and going back in menu w/ Deceptious mods.
+	  "LocKey#6820"    // Actually drinking or talking to your drinking partner.
+	];
+	this.lover = [
+	  "LocKey#34479",   // Judy @ her home and a few other places in the main story
+      "Judy",           // Judy @ your house
+	  "River",          // River @ his home and a few other places in the main story
```

Comment:
Are these ones from the base game or Deceptious mod as well?

Reply:
Yeah, I think you are right, his mod may be overriding these. I think I am going to have to uninstall his stuff to find the default ones to add to the list.

Reply:
`Check` from EdgerunnerInteractionChecker has some logging commented, you can replace it with FTLog maybe (or just with custom logger from utils/Log.reds) and check which lockey is actually there when you interact with the characters.

Reply:
Can confirm that the Lockey is the exact same both with and without the mods. I already had been playing the whole time with a logger specifically to investigate interactions for some other mods I am working on. I guess even in non-English locales, some LocKeys themselves are English. Makes sense if you think about it. I further confirmed this because there are other times in the game where "Judy" is the Lockey used for Judy. Completely divorced from any mods interactions (in this case when you talk to her outside of Maika's office).

Reply:
Also checked river for good measure since he is on my V's rotation as well:

Teleporting to, and then talking to River as one of our current confirmed lovers:
```
[2024-10-06 00:37:48 UTC-04:00] [38624] [Edgerunner] Remove BaseStatusEffect.NewPrePsychosisEffectNormalFx effect from player
[2024-10-06 00:38:05 UTC-04:00] [56260] [Edgerunner] locKey: 
[2024-10-06 00:38:45 UTC-04:00] [55260] [Edgerunner] locKey: River
[2024-10-06 00:38:45 UTC-04:00] [55260] [Edgerunner] > RemoveHumanityDamage: current 29, cost  3, after removal: 26
[2024-10-06 00:38:45 UTC-04:00] [55260] [Edgerunner] ! Lover, humanity restored
[2024-10-06 00:38:45 UTC-04:00] [55260] [Edgerunner] INVALIDATE: total 130, current 104, damage 26 | threshold 20 | from mods 0 | light visuals false
[2024-10-06 00:38:45 UTC-04:00] [55260] [Edgerunner] ? Check for neuroblockers tag: false
[2024-10-06 00:38:45 UTC-04:00] [55260] [Edgerunner] ? Stop everything 
[2024-10-06 00:38:45 UTC-04:00] [55260] [Edgerunner] ? Stop pre-psychosis
[2024-10-06 00:38:45 UTC-04:00] [55260] [Edgerunner] !!! Fx New - Stop Pre Psychosis
[2024-10-06 00:38:45 UTC-04:00] [55260] [Edgerunner] ? Stop psychosis
[2024-10-06 00:38:45 UTC-04:00] [55260] [Edgerunner] !!! Fx New - Psychosis stop
[2024-10-06 00:38:45 UTC-04:00] [55260] [Edgerunner] + Stop status_bleeding vfx
[2024-10-06 00:38:45 UTC-04:00] [55260] [Edgerunner] Sfx - Cancel cycled fx
[2024-10-06 00:38:45 UTC-04:00] [55260] [Edgerunner] Sfx - Cancel other fxes
[2024-10-06 00:38:45 UTC-04:00] [55260] [Edgerunner] + Stop fx_damage_high vfx
[2024-10-06 00:38:45 UTC-04:00] [55260] [Edgerunner] + Stop personal_link_glitch vfx
[2024-10-06 00:38:45 UTC-04:00] [55260] [Edgerunner] + Stop hacking_glitch_low vfx
[2024-10-06 00:38:45 UTC-04:00] [55260] [Edgerunner] + Stop ui_gmpl_perk_edgerunner sfx
[2024-10-06 00:38:45 UTC-04:00] [55260] [Edgerunner] Prevention - Cancel everything
[2024-10-06 00:38:45 UTC-04:00] [55260] [Edgerunner] Victims - Cancel spawns
[2024-10-06 00:38:45 UTC-04:00] [55260] [Edgerunner] ? Stop post-psychosis
[2024-10-06 00:38:45 UTC-04:00] [55260] [Edgerunner] !!! Fx New - Post Psychosis stop
[2024-10-06 00:38:46 UTC-04:00] [55260] [Edgerunner] Remove BaseStatusEffect.NewPostPsychosisEffectLightFx effect from player
[2024-10-06 00:38:46 UTC-04:00] [55260] [Edgerunner] Remove BaseStatusEffect.NewPsychosisEffectLightFx effect from player
[2024-10-06 00:38:46 UTC-04:00] [55260] [Edgerunner] Remove BaseStatusEffect.NewPrePsychosisEffectLightFx effect from player
[2024-10-06 00:38:46 UTC-04:00] [55272] [Edgerunner] Remove BaseStatusEffect.NewPostPsychosisEffectNormalFx effect from player
[2024-10-06 00:38:46 UTC-04:00] [55272] [Edgerunner] Remove BaseStatusEffect.NewPsychosisEffectNormalFx effect from player
[2024-10-06 00:38:46 UTC-04:00] [55272] [Edgerunner] Remove BaseStatusEffect.NewPrePsychosisEffectNormalFx effect from player
```